.prettierignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
There is no need to add `node_modules/` to this file; it's ignored by default.

Reply:
Also, you still haven't addressed the https://github.com/metarhia/common/pull/243#discussion_r247047923; you must specify the files that should not be formatted by prettier in this file.